5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
+	echo "CONTAINER[${HOSTNAME:-}] `date +%T`     $@"
```

Comment:
I'd rather use something more precise, like `date --iso-8601=seconds` or even `date --iso-8601=ns`, for cases we want to see the logs from history and for cases we want to compare logs from different containers when finding race conditions. Might be unnecessary in many cases, but might come handy once..